### PR TITLE
[docs] page for using Modin with Ray

### PR DESCRIPTION
**Pull request by devin-petersohn**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/modin/index.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+   ray.init()
+   df = pd.read_parquet("s3://my-bucket/big.parquet")
+
+You can use Modin on Ray with your laptop or cluster. In this document,
+we show instructions for how to set up a Modin compatible Ray cluster
+and connect Modin to Ray.
+
+**Note: in previous versions of Modin you had to initialize Ray before
+importing Modin. This is no longer the case**
+
```
Comment (richardliaw) [inline doc/source/modin/index.rst:23]:
```suggestion
.. note:: In previous versions of Modin, you had to initialize Ray before
importing Modin. This is no longer the case.

```

**Comment by richardliaw** [comment]
hey @devin-petersohn do you mind if i push to your branch?

**Comment by devin-petersohn** [comment]
@richardliaw Go for it.

File: doc/source/modin/index.rst [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+   import ray
+
+   ray.init()
+   df = pd.read_parquet("s3://my-bucket/big.parquet")
+
+You can use Modin on Ray with your laptop or cluster. In this document,
+we show instructions for how to set up a Modin compatible Ray cluster
+and connect Modin to Ray.
+
+.. note:: In previous versions of Modin, you had to initialize Ray before importing Modin. This is no longer the case.
```
Comment (rkooo567) [inline doc/source/modin/index.rst:21]: Mention the specific version here?

Comment (devin-petersohn) [reply doc/source/modin/index.rst]: Done, thanks!

**Review by richardliaw — APPROVED** [review]

**Pull request merged**